cmd/eksctl-anywhere/cmd/generateclusterconfig.go

```diff
@@ -194,6 +194,7 @@ func generateClusterConfig(clusterName string) error {
 			v1alpha1.WorkerNodeConfigCount(1),
 			v1alpha1.WorkerNodeConfigName(constants.DefaultWorkerNodeGroupName),
 		)
+		clusterConfigOpts = append(clusterConfigOpts, v1alpha1.WithCPUpgradeRolloutStrategy(1, 0), v1alpha1.WithWorkerMachineUpgradeRolloutStrategy(1, 0))
```

Comment:
I thought this was going to be optional and not a default?

Reply:
It is optional, but the default values do show up in cluster config. If it is specifically removed, a warning to the effect of 
`ControlPlaneConfiguration: UpgradeRolloutStrategy not specified in cluster config. CAPI will default to 'RollingUpdate' with maxSurge=1`
or 
`WorkerNodeGroupConfigurations: UpgradeRolloutStrategy not specified in cluster config. CAPI will default to 'RollingUpdate' with maxSurge=1 and maxUnavailable=0`
is printed and cli moves on.

Reply:
Can we not default to it showing up? This would then truly be an optional field, unless we want it to be specified?

Reply:
I was only showing it for Tinkerbell provider because someone would highly likely want to change the default values that match with CAPI. But it might not be a bad idea to make it consistent with other providers and hide it, then communicate to users via docs. Will change for now to not show, we can discuss further